engine/src/main/java/com/badlogic/gdx/graphics/g2d/Animation.java

```diff
@@ -0,0 +1,216 @@
+/*Taken from the LibGDX source code from https://github.com/libgdx/libgdx on 03/11/2018*/
```

Comment:
Sure, I see no reason to reinvent the wheel we can just ~steal~ borrow it from someone else :D